tests/system-broker/tests/auth_test.go

```diff
@@ -95,6 +115,66 @@ func TestSystemBrokerAuthentication(t *testing.T) {
 	})
 }
 
+func TestCallingORDServiceWithCert(t *testing.T) {
+	ORDContext, err := pkg.NewTestContext(config)
```

Comment:
Instead of `ORDContext` this can simply be `testContext`, or `testCtx`, as it is not entirely ORD specific.